compiler/control/OptionsUtil.hpp

```diff
@@ -208,6 +208,7 @@ class OptionSet {
 //
 enum CodeCacheKind {
     DEFAULT_CC = 0,
+    TRANSIENT_CODE_CC,
```

Comment:
Pleas add a short comment indicating what this new CodeCacheKind is supposed to be used for.